index.Rmd

```diff
@@ -136,9 +136,9 @@ p1 <- crime_income %>%
 This figure is a map of crime by type in 2017
 
 ```{r}
-p1 + geom_point(crime_income, aes(x = crime_date, y = Assault))
+p1 + geom_point(crime_income, aes(x = crime_date, y = Assault))  
```

Comment:
You reference this object, crime_income, but I can't actually see where it's been created. Should this maybe read crime_income2017?